fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -707,12 +744,12 @@ ACTOR Future<Void> runClient(GlobalTagThrottler* globalTagThrottler,
 }
 
 ACTOR template <class Check>
-Future<Void> monitor(GlobalTagThrottler* globalTagThrottler, Check check) {
+Future<Void> monitorActor(GlobalTagThrottler* globalTagThrottler, Check check) {
 	state int successes = 0;
 	loop {
 		wait(delay(1.0));
 		if (check(*globalTagThrottler)) {
-			if (++successes == 3) {
+			if (++successes == 10) {
```

Comment:
Is `10` a special number (or an arbitrary threshold)? Can you add a comment here?

Reply:
10 is an arbitrary threshold, I added a comment to explain this line.